### ISSUE TITLE
Jitpack v1.0.1 build failed

### ISSUE THREAD

**Issue by martinBoscNomad** [amitshekhariitbhu/PRDownloader#215]
I'm unable to use this library since today

I used to import `com.mindorks.android:prdownloader:0.6.0' but it's not found since yesterday in maven/jitpack

i tried  `com.github.amitshekhariitbhu:PRDownloader:1.0.1` but i got this error message : 
`Failed to resolve: com.github.amitshekhariitbhu:PRDownloader:1.0.1`

In jitpack, here is the log:
https://jitpack.io/com/github/amitshekhariitbhu/PRDownloader/1.0.1/build.log
BUILD FAILED

**Comment by amitshekhariitbhu** [comment]
Have you configured the gradle for Jitpack?

**Comment by amitshekhariitbhu** [comment]
![IMG_7723](https://github.com/user-attachments/assets/1aa3c3cd-fc17-4853-b448-ae77b73627aa)

**Comment by martinBoscNomad** [comment]
Yes, I already added jitpack.io in gradle conf for other github libs import.

But with the new version 1.0.1, when I try to sync project with gradle files i got:
<img width="504" alt="Capture d’écran 2024-08-22 à 09 47 35" src="https://github.com/user-attachments/assets/57a9120a-7f78-458b-b4a7-f283aa8a0fba">
<img width="527" alt="Capture d’écran 2024-08-22 à 09 47 40" src="https://github.com/user-attachments/assets/6b82bd50-c9dc-40bc-8326-83dac62931ef">

And I think it's because jitpack build failed, if you look at jitpack logs  https://jitpack.io/com/github/amitshekhariitbhu/PRDownloader/1.0.1/build.log :
<img width="1174" alt="Capture d’écran 2024-08-22 à 09 49 16" src="https://github.com/user-attachments/assets/ee601333-79b0-4e16-a50d-9b1414cd9ed3">

I tried to get the jitpack generated jar file and import it, and nothing works with it.

**Comment by WestFR** [comment]
Same error @martinBoscNomad

**Comment by amitshekhariitbhu** [comment]
Refer this: https://github.com/amitshekhariitbhu/PRDownloader/issues/209

**Comment by danielpearsports** [comment]
Looks like Jitpack rebuilt the package on Feb 10th with the following error:

`1 errors, 0 warnings
Lint found fatal errors while assembling a release target.

To proceed, either fix the issues identified by lint, or modify your build script as follows:
...
android {
    lintOptions {
        checkReleaseBuilds false
        // Or, if you prefer, you can continue to check for errors in release builds,
        // but continue the build even when errors are found:
        abortOnError false
    }
}
...

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:lintVitalRelease'.
> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction
         > Lint found fatal errors while assembling a release target.
`

**Comment by Berkay92** [comment]
Found an alternative version of this library without having those issues;
`implementation 'com.github.alexto9090:PRDownloader:1.0'`

Check out [its repo](https://github.com/alexto9090/PRDownloader)

**Comment by amitshekhariitbhu** [comment]
It is fixed: Please try the lastest `implementation 'com.github.amitshekhariitbhu:PRDownloader:1.0.2'`

**Comment by Alhaadi-IDB** [comment]
> It is fixed: Please try the latest `implementation 'com.github.amitshekhariitbhu:PRDownloader:1.0.2'`

face same issue, after update the Gradle version, after update to latest version "1.0.2", . problem solved. 

Thanks. @amitshekhariitbhu
